traffic_portal/app/src/common/modules/table/servers/TableServersController.js

```diff
@@ -155,6 +155,11 @@ var TableServersController = function(tableName, servers, filter, $scope, $state
 			field: "offlineReason",
 			hide: true
 		},
+		{
```

Comment:
nit: One other thing in TP, under the column it shows correct spacing but when hovered over it, spacing between , and number disappears
![image](https://user-images.githubusercontent.com/22248619/186255797-e6f5fb06-480c-44f0-87d8-459cb62f9bda.png)